openslides_backend/action/actions/meeting/create.py

```diff
@@ -156,6 +156,7 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         )
         instance["list_of_speakers_countdown_id"] = action_results[0]["id"]  # type: ignore
         instance["poll_countdown_id"] = action_results[1]["id"]  # type: ignore
+        instance["projector_countdown_warning_time"] = 1
```

Comment:
If this is merged as a hotfix, please create an issue, that this value must be 0.

Reply:
Add issue https://github.com/OpenSlides/openslides-backend/issues/661

Reply:
Please wait for PR https://github.com/OpenSlides/openslides-backend/pull/662 / Issue  https://github.com/OpenSlides/openslides-backend/issues/661. It is ready for code review and resolves the problem mentioned above.

Reply:
> Please wait for PR #662 / Issue #661. It is ready for code review and resolves the problem mentioned above.

Done, you could finis this